applications/GeoMechanicsApplication/tests/test_thermal_element/test_thermal_line_element/README.md

```diff
@@ -0,0 +1,17 @@
+# Test Cases for Thermal Line Element temperature
+
+**Author:** [Mohamed Nabi](https://github.com/mnabideltares)
+
+**Source files:** [Thermal line element with fixed temperature](https://github.com/KratosMultiphysics/Kratos/tree/master/applications/GeoMechanicsApplication/tests/test_thermal_element/test_thermal_line_element)
+
+## Case Specification
+In this thermal test case, a 3 m deep soil is considered, with everywhere set to 0 degrees then a sudden jump at the top and bottom boundary temperatures are given. These are -10 degrees at bottom and 100 degrees at the top boundaries. The simulation spans 1000 days to allow for a transition from an exponential to a linear temperature profile between the two sides. This test is conducted for various configurations, including 2D2N, 2D3N, 2D4N, 2D5N, 3D2N and 3D3N line elements. The temperature distribution along the depth is then evaluated with its own result.
+The boundary conditions are shown below:
+
+<img src="../documentation_data/test_thermal_line_element.svg" alt="Visualization of the Boundary conditions" title="Visualization of the Boundary conditions" width="600">
+
+## Results
+
+The picture below illustrates the temperature contours resulting from the simulation (as an example the 2D3N test is shown below).
```

Comment:
In my opinion it would be helpful to add a sentence to the results section that explains that the assert you are doing in the test itself (checking the $-10 + (100 -- 10)/3 = 26.66667$ on the second node).

Reply:
In addition to Richard's comment, I believe it would be helpful to explicitly state in the text that the displayed temperature contour plot was created for the final time step.